write-fonts/src/graph.rs

```diff
@@ -82,9 +84,9 @@ impl ObjectStore {
 
 #[derive(Debug)]
 pub struct Graph {
```

Comment:
I think perhaps this one deserves a comment, even if it's just a "go read here to understand me"? - Graph of objects, nodes, and order with some invalid bits is extremely general and has non-obvious interpretation